cli/src/commands/invocations/cancel.rs

```diff
@@ -38,18 +42,26 @@ pub struct Cancel {
     /// Ungracefully kill the invocation and its children
     #[clap(long)]
     pub(super) kill: bool,
+    /// Limit the number of fetched invocations
+    #[clap(long, default_value = "DEFAULT_BATCH_INVOCATIONS_OPERATION_LIMIT")]
```

Comment:
I think this is a problem here. Now the default value is the literal string "DEFAULT_BATCH_INVOCATIONS_OPERATION_LIMIT" which is an invalid usize. 


```suggestion
    #[clap(long, default_value_t = DEFAULT_BATCH_INVOCATIONS_OPERATION_LIMIT)]
```

Reply:
Uh... why didn't i catch it nor compilation failed? :thinking:

Reply:
unfortunately this will only happen in runtime if you didn't pass a value.

Reply:
uhhhh ok i realized, when i tested this i was always testing passing the --limit myself... thanks for catching that!